docs/products/clickhouse/howto/get-started/connect-to-service.rst

```diff
@@ -0,0 +1 @@
+.. include:: /docs/products/clickhouse/howto/connect.rst
```

Comment:
Looks like this is not loading the content from connect.rst file as expected. I see a blank screen.